go.mod

```diff
@@ -165,28 +142,25 @@ require (
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
 	golang.org/x/tools v0.1.9
 	google.golang.org/api v0.62.0
-	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa
-	google.golang.org/grpc v1.43.0
-	google.golang.org/protobuf v1.27.1 // indirect
+	google.golang.org/genproto v0.0.0-20220329172620-7be39ac1afc7
+	google.golang.org/grpc v1.45.0
+	google.golang.org/protobuf v1.28.0
 	gopkg.in/inf.v0 v0.9.1
-	gopkg.in/jcmturner/aescts.v1 v1.0.1 // indirect
-	gopkg.in/jcmturner/dnsutils.v1 v1.0.1 // indirect
-	gopkg.in/jcmturner/goidentity.v3 v3.0.0 // indirect
 	gopkg.in/jcmturner/gokrb5.v7 v7.5.0
-	gopkg.in/jcmturner/rpc.v1 v1.1.0 // indirect
 	gopkg.in/mgo.v2 v2.0.0-20160818020120-3f83fa500528
 	gopkg.in/yaml.v2 v2.4.0
 	gotest.tools v2.2.0+incompatible
-	gotest.tools/gotestsum v0.6.0
+	gotest.tools/gotestsum v1.7.0
 	howett.net/plist v1.0.0
-	k8s.io/api v0.23.0
-	k8s.io/apimachinery v0.23.0
-	k8s.io/client-go v0.23.0
+	k8s.io/api v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/client-go v0.23.1
 	kernel.org/pub/linux/libs/security/libcap/cap v1.2.57
 )
 
 require (
 	github.com/elastic/elastic-agent-libs v0.1.1
+	github.com/elastic/elastic-agent-shipper v0.2.0
```

Comment:
When we eventually try to import the queue from beats into the shipper this is going to cause an import cycle. We could potentially move the API definitions to a separate repo like https://github.com/elastic/elastic-agent-client but there might be other ways to avoid this.

Reply:
I'd be inclined to just copy the API definitions for now. Even doing the "right" thing and linking to some external for the API would still require a PR when the spec is updated, and since this output is mostly about bootstrapping the data flow for the shipper work I think just including the definitions for now is simple and relatively harmless.

Reply:
Yeah I agree that we should prioritize being able to make changes quickly initially.